client/src/components/ContentManagement/ContentManagementSteps.tsx

```diff
@@ -1093,116 +1100,118 @@ const ContentManagementSteps: FC<Props> = () => {
+                        ...oldForm,
+                        incorrectMessage,
+                      }))}
+                      explanation={
+                                  (newQuizFormValues as MultipleChoiceQuizForm)
+                                    .explanation
+                              }
+                      handleChangeExplanation={(explanation) => setNewQuizFormValues((oldForm) => ({
+                        ...oldForm,
+                        explanation,
+                      }))}
+                    />
+                  ) : newQuizFormValues.type === 'tf' ? (
+                    <TFQuizForm
+                      isTrue={
+                              (newQuizFormValues as TrueFalseQuizForm)
+                                .isTrue
+                          }
+                      handleChangeIsTrue={(isTrue) => setNewQuizFormValues((oldForm) => ({
+                        ...oldForm,
+                        isTrue,
+                      }))}
+                      correctMessage={
+                              (newQuizFormValues as TrueFalseQuizForm)
+                                .correctMessage
+                          }
+                      handleChangeCorrectMessage={(
+                        correctMessage,
+                      ) => setNewQuizFormValues((oldForm) => ({
+                        ...oldForm,
+                        correctMessage,
+                      }))}
+                      incorrectMessage={
+                              (newQuizFormValues as TrueFalseQuizForm)
+                                .incorrectMessage
+                          }
+                      handleChangeIncorrectMessage={(
+                        incorrectMessage,
+                      ) => setNewQuizFormValues((oldForm) => ({
+                        ...oldForm,
+                        incorrectMessage,
+                      }))}
+                      explanation={
+                              (newQuizFormValues as TrueFalseQuizForm)
+                                .explanation
+                          }
+                      handleChangeExplanation={(explanation) => setNewQuizFormValues((oldForm) => ({
+                        ...oldForm,
+                        explanation,
+                      }))}
+                    />
+                  ) : newQuizFormValues.type === 'qa' ? (
+                    <QAQuizForm
+                      explanation={
+                              (newQuizFormValues as QuestionAnswerQuizForm)
+                                .explanation
+                          }
+                      handleChangeExplanation={(explanation) => setNewQuizFormValues((oldForm) => ({
+                        ...oldForm,
+                        explanation,
+                      }))}
+                    />
+                  ) : (
+                    null
+                  ))}
 
                   <Button
                     color="secondary"
```

Comment:
This file is over 1000 lines. Is there a way to break it up into smaller files? @Tymotex